server/engine/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -1824,12 +1826,15 @@ private ProcedureName parseProcedureName(CobolParser.ProcedureNameContext proced
     return new ProcedureName(targetName, sectionName);
   }
 
-  private void throwException(String wrongToken, @NonNull Locality locality, String message) {
+  private void throwException(
+      String wrongToken, @NonNull Locality locality, MessageTemplate messageTemplate) {
```

Comment:
Wouldn't it be easier to add `%s` to those three messages and use MessageTemplate.of just like anywhere else?